code/game/turfs/closed/minerals.dm

```diff
@@ -32,13 +32,19 @@
 
 /turf/closed/mineral/Initialize(mapload)
 	. = ..()
-	var/matrix/M = new
-	M.Translate(-4, -4)
```

Comment:
@LemonInTheDark I'm not home to test, but is this different from pixel_x or whatever somehow?

Reply:
somewhat, also transforms is just a less touched tool, so it's much less likely to be overriden.
could maybe use base pixel y? see what happens?

Reply:
Nothing seems to use base_pixels for this

Reply:
oh is that mob shit

Reply:
prolly